poi-radio/src/lib.rs

```diff
@@ -0,0 +1,191 @@
+use async_graphql::{Error, ErrorExtensions, SimpleObject};
+use autometrics::autometrics;
+use ethers_contract::EthAbiType;
+use ethers_core::types::transaction::eip712::Eip712;
+use ethers_derive_eip712::*;
+use once_cell::sync::OnceCell;
+use prost::Message;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::HashMap,
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc, Mutex as SyncMutex,
+    },
+};
+use tokio::signal;
+use tracing::error;
+
+use graphcast_sdk::{
+    graphcast_agent::GraphcastAgentError,
+    graphql::{client_graph_node::get_indexing_statuses, QueryError},
+};
+use graphcast_sdk::{
+    graphcast_agent::{message_typing::GraphcastMessage, GraphcastAgent},
+    graphql::client_network::query_network_subgraph,
+    networks::NetworkName,
+    BlockPointer,
+};
+
+use crate::operator::attestation::AttestationError;
+
+pub mod config;
+pub mod graphql;
+pub mod metrics;
+pub mod operator;
+pub mod server;
+pub mod state;
+
+pub type MessagesVec = OnceCell<Arc<SyncMutex<Vec<GraphcastMessage<RadioPayloadMessage>>>>>;
```

Comment:
This isn't used anymore, could delete in a future PR. Aside from that, I wonder why this file (and a few others living in poi-radio/src/) is recognized as a new file instead of being recognized as rename without changes?

Reply:
i think it's because it changes folders, so it's technically a "new" file as far as the path is concerned

Reply:
I don't think that's the case. For instance in this screenshot you can see that the path changing is considered as renaming.
![Screenshot 2023-05-31 at 2 12 26 PM](https://github.com/graphops/poi-radio/assets/60078528/382bba03-b202-4ded-b6e0-e03c8a2a8992)

Just want to be certain that poi-radio/src code was moved properly such that github could understand they were existing files. I think these files should all be recognized as renames instead of new files... 
![Screenshot 2023-05-31 at 2 14 29 PM](https://github.com/graphops/poi-radio/assets/60078528/e3340e6f-5259-47fb-865e-8d025bd624d2)

Reply:
hmm not sure then tbh :( github being weird

Reply:
took a look at the branch and I think that's because the old files are still there. Maybe remove all the previous code from the root `src` which currently still contains those files that should be registered as renamed (`lib.rs`, `main.rs`, `/operator` :')

Reply:
ahh you're right, the `src/` is still there 🤦🏻‍♂️  smh. Should be good now